buildkite/build_release.sh

```diff
@@ -34,9 +34,9 @@ aws --profile ddo s3 sync dmd-$BRANCH-$TODAY/ s3://downloads.dlang.org/nightlies
 
 # create redirects from dmd-$BRANCH/* to dmd-$BRANCH-2018-10-12/*
 for file in $(ls dmd-$BRANCH-$TODAY); do
-    echo -n | aws --profile ddo s3 cp - s3://downloads.dlang.org/nightlies/dmd-$BRANCH/$file --acl public-read --cache-control max-age=604800 --website-redirect http://downloads.dlang.org/nightlies/dmd-$BRANCH-$TODAY/$file
+    echo -n | aws --profile ddo s3 cp - s3://downloads.dlang.org/nightlies/dmd-$BRANCH/$file --acl public-read --cache-control max-age=604800 --website-redirect https://downloads.dlang.org/nightlies/dmd-$BRANCH-$TODAY/$file
```

Comment:
Hum we're not pointing to anymore ?

Reply:
Hmm, yes. That's right. The downloads.dlang.org record is no longer usable as an S3 bucket.